### PR TITLE
Homeapis

### PR DESCRIPTION
**Pull request by Chandini-212634**

### CODE REVIEW

File: src/main/resources/application.properties
Comment (shijukbabu-ust) [inline src/main/resources/application.properties:1]: Do not change this file. Please revert and commit.

**Review by shijukbabu-ust — CHANGES_REQUESTED** [review]

**Comment by shijukbabu-ust** [comment]
Resolve conflicts as well.

File: src/main/resources/application.properties
Comment (Chandini-212634) [reply src/main/resources/application.properties:1]: resolved

File: src/main/java/com/ust/sourcecourse/configuration/repository/SourceColumnRepository.java [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-import org.springframework.stereotype.Repository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 import com.ust.sourcecourse.configuration.entity.SourceColumn;
 
-@Repository
 public interface SourceColumnRepository extends JpaRepository<SourceColumn, Long> {
 
+	@Query("SELECT sc FROM SourceColumn sc JOIN st.tags c WHERE c = LOWER(:tag)")
```
[the remark this answers is not in the document]
Comment (Chandini-212634) [reply src/main/java/com/ust/sourcecourse/configuration/repository/SourceColumnRepository.java]: resolved

**Comment by shijukbabu-ust** [comment]
Pull from dev and resolve conflict.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,9 +1,9 @@
-server.port=8080
+server.port=8081
 server.servlet.context-path=/sourcecourse
-spring.datasource.url=jdbc:mysql://localhost:3306/sourcecourse?useSSL =true
+spring.datasource.url=jdbc:mysql://localhost/sourcecource?useSSL =true
```
Comment (shijukbabu-ust) [inline src/main/resources/application.properties:3]: DB name should be `sourcecourse`

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,9 +1,9 @@
-server.port=8080
+server.port=8081
```
[the remark this answers is not in the document]
Comment (Chandini-212634) [reply src/main/resources/application.properties]: resolved

File: src/main/resources/application.properties (hunk 2 above)
Comment (Chandini-212634) [reply src/main/resources/application.properties]: resolved

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 server.servlet.context-path=/sourcecourse
-spring.datasource.url=jdbc:mysql://localhost:3306/sourcecourse?useSSL =true
+spring.datasource.url=jdbc:mysql://localhost/sourcecource?useSSL =true
 spring.datasource.username=root
 spring.datasource.password=Root@1234
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
```
[the remark this answers is not in the document]
Comment (Chandini-212634) [reply src/main/resources/application.properties]: resolved

**Review by shijukbabu-ust — APPROVED** [review]

**Pull request merged**